packages/frontend/src/components/claim-rewards/ClaimRewards.vue

```diff
@@ -53,28 +57,34 @@ const claimRewardsParams = computed(() => {
         }));
 });
 
-const aggregatedClaims = computed(() => {
+const mergedClaims = computed(() => {
     if (!claims.value || !account.value.address) return [];
 
     const collator = new Intl.Collator();
 
     return Object.values(
-        claims.value.reduce((accumulator: Record<string, Claim>, claim) => {
+        claims.value.reduce((acc: Record<string, MergedClaim>, claim) => {
             // avoid mutating the original claims
-            const clonedClaim = { ...claim };
+            const cloned = { ...claim };
```

Comment:
I honestly think we might want to avoid this here and instead set this directly at line 72, using the `claim` object to then update the values at lines 78 and 79. The idea is that in case the `acc[claim.token.address]` is already there we avoid the unnecessary cloning.

Reply:
Fixed in e8510dd98aaa780e17b03ee1b550a4bc6c4213de.